### ISSUE TITLE
About the metrics

### ISSUE THREAD

**Issue by zhuhrun** [dvlab-research/VisionReasoner#23]
Hello, excellent work.

I seem to be encountering some issues. I trained the model on an A800-80GB GPU. 
During my training, the metric `“critic/rewards/mean”` appears difficult to exceed `5`. I'm curious to know what values the metrics `“critic/rewards/mean”` and `“critic/rewards/min”` typically reach in your training?May i have your training log.

```python
# reward
"critic/rewards/mean": torch.mean(sequence_reward).detach().item(),
"critic/rewards/max": torch.max(sequence_reward).detach().item(),
"critic/rewards/min": torch.min(sequence_reward).detach().item(),
``` 

Thank you for your prompt reply.
